.evergreen/config.yml

```diff
@@ -1381,6 +1380,10 @@ tasks:
       - command: subprocess.exec
         params:
           binary: bash
+          # Set the GO_VERSION to empty string to use the Go installation in the
+          # PATH.
+          env:
+            GO_VERSION: ""
```

Comment:
Shouldn't this be pointing to 1.21 now?

Reply:
Setting `GO_VERSION=""` causes the script to use the `go` binary in PATH, which is Go 1.22 currently. My goal was to not have another repetition of the Go version in the Evergreen config.

Reply:
Ah, I see now